source/type/pt_data.cpp

```diff
@@ -44,6 +44,7 @@ www.navitia.io
 #include "type/multi_polygon_map.h"
 #include "type/commercial_mode.h"
 #include "type/physical_mode.h"
+#include "type/route_point.h"
```

Comment:
it is alreeady include here ? https://github.com/CanalTP/navitia/pull/3401/files#diff-adeca346ad47a2fe4760401a823171620dcf4bb21b22a2729e0de603845cb230R36